opm/parser/eclipse/EclipseState/Schedule/Well/Well.hpp

```diff
@@ -624,21 +632,21 @@ class Well {
     double solvent_fraction;
     bool has_produced = false;
     bool prediction_mode = true;
-
+    double productivity_index{ std::numeric_limits<double>::lowest() };
```

Comment:
Could this be a `std::optional<double>`instead of using a magic uninitialized value?

Reply:
> Could this be a `std::optional<double>`

Good idea.  I've made this change in the latest update.